src/game/server/gamecontext.h

```diff
@@ -355,6 +355,7 @@ class CGameContext : public IGameServer
 	bool RateLimitPlayerMapVote(int ClientID) const;
 
 	void OnUpdatePlayerServerInfo(char *aBuf, int BufSize, int ID) override;
+	const char *ScoreKind() override;
```

Comment:
Please use different names for methods returning different types. It is confusing to have `const char *CGameContext::ScoreKind()` and `CServerInfo::EClientScoreKind IGameController::ScoreKind(int ClientID)`, even though they have different args.

```suggestion
	const char *ScoreKindStr() override;
```